nekoyume/Assets/_Scripts/UI/Menu.cs

```diff
@@ -444,8 +444,10 @@ private void PlayTutorial()
                 if (recipeRow is null)
                 {
                     Debug.LogError("EquipmentItemRecipeSheet is empty");
+                    return;
                 }
-                else if (States.Instance.CurrentAvatarState.inventory.HasItem(recipeRow.ResultEquipmentId))
+
+                if (!States.Instance.CurrentAvatarState.inventory.HasItem(recipeRow.MaterialId))
```

Comment:
if `recipeRow` is null, it will be throw exception

Reply:
```
if (recipeRow is null)
{
    Debug.LogError("EquipmentItemRecipeSheet is empty");
    return;
}
```
`recipeRow` variable is already filtered at this moment.